test/Microsoft.AspNet.Mvc.Core.Test/Filters/ResponseCacheFilterTest.cs

```diff
@@ -221,14 +204,16 @@ public void IsOverridden_ReturnsTrueForAllButLastFilter()
         {
             // Arrange
             var caches = new List<IFilter>();
-            caches.Add(new ResponseCacheAttribute());
-            caches.Add(new ResponseCacheAttribute());
+            caches.Add(new ResponseCacheFilter(
+                duration: 0, location: ResponseCacheLocation.Any, noStore: false, varyByHeader: null));
+            caches.Add(new ResponseCacheFilter(
+                duration: 0, location: ResponseCacheLocation.Any, noStore: false, varyByHeader: null));
 
             var context = GetActionExecutingContext(caches);
 
             // Act & Assert
-            Assert.True((caches[0] as ResponseCacheAttribute).IsOverridden(context));
-            Assert.False((caches[1] as ResponseCacheAttribute).IsOverridden(context));
+            Assert.True((caches[0] as ResponseCacheFilter).IsOverridden(context));
```

Comment:
nit: instead of this, we should do something like Assert.IsType<ResponseCacheFilter> and then use the instance to check if it is overriden.